src/dgipy/vcf.py

```diff
@@ -0,0 +1,202 @@
+"""Provides methods for annotating VCF with DGIdb data"""
+
+import contextlib
+from itertools import groupby
+
+import pandas as pd
+import pysam
+import requests
+from tqdm import tqdm
+
+import dgipy
+
+# Sample usage: import vcf
+#               data = vcf.annotate('link/to/file',chr='chr#')
+
+
+# TODO: Probably need another class as a wrapper object rather than putting it all in a list
+# Class would have analogous display methods but also allow access to individual GeneResults
+class GeneResult:
+    """A gene result from original VCF
+
+    Attributes
+    ----------
+    gene : str
+        The name of the gene
+    records : list
+        The mapped records from original VCF
+    interactions : pd.DataFrame
+        The interactions obtained from DGIdb
+
+    Methods
+    -------
+    search_interactions(gene: str) -> pd.DataFrame
+        Search drug-gene interactions for given gene in DGIdb
+
```

Comment:
Should delete this -- it can be autogenerated in sphinx

Reply:
removed